add step for tagging subnets (#543)
awslabs/karpenter: website/content/en/docs/getting-started/_index.md

Additionally, the configuration file sets up [IAM Roles for Service Accounts](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html) (IRSA), which grants Karpenter permissions like launching instances.

### Tag Subnets

Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
Retreive the subnet IDs and tag them with the cluster name.

> 

Reviewer: ```suggestion
```

Additionally, the configuration file sets up [IAM Roles for Service Accounts](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html) (IRSA), which grants Karpenter permissions like launching instances.

### Tag Subnets

Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
Retreive the subnet IDs and tag them with the cluster name.

> The subnets should be automatically tagged.

Reviewer: ```suggestion
```

necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
of communicating with identity providers. 

> ### Tag Subdomains

Reviewer: Tag Subnets

> ### Tag Subdomains

Reviewer 2: ```suggestion
### Tag Subnets
```

> Karpenter expects subnets to be tagged with the cluster name for autodiscovery.

Reviewer: Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.

```...```

> Karpenter expects subnets to be tagged with the cluster name for autodiscovery.

Reviewer 2: ```suggestion
Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
```

Retreive the subnet IDs and tag them with the cluster name.

> Note: If you have a cluster with version 1.18 or below you can skip this step.

Reviewer: We're on 1.21 and 1.18 is officially out of support. I'd leave this out.

> Note: If you have a cluster with version 1.18 or below you can skip this step.

Reviewer 2: ```suggestion
```

The subnets should be automatically tagged.

```bash
SUBNET_IDS=$(aws cloudformation describe-stacks \
    --stack-name eksctl-${CLUSTER_NAME}-cluster \
    --query 'Stacks[].Outputs[?OutputKey==`SubnetsPrivate`].OutputValue' \
    --output text)

> 

Reviewer: I'd drop this newline

> 

Reviewer 2: ```suggestion
```
